src/Umbraco.Web.UI.Client/src/common/services/search.service.js

```diff
@@ -66,11 +66,12 @@ angular.module('umbraco.services')
                     throw "args.term is required";
                 }
 
-                return entityResource.search(args.term, "Document", args.searchFrom, args.canceler, args.dataTypeKey)
-                    _.each(data, function (item) {
-                        data.forEach(item => searchResultFormatter.configureContentResult(item));
-                        return data;
+                return entityResource.search(args.term, 'Document', args.searchFrom, args.canceler, args.dataTypeKey).then(function (data) {
+                    data.forEach(function (item) {
+                        return searchResultFormatter.configureContentResult(item);
                     });
+                    return data;
+                });
             },
```

Comment:
@mcl-sz yes it seems in was changed in this PR https://github.com/umbraco/Umbraco-CMS/pull/8534

Alternatively it could use ES6 arrow syntax as with the other functions.

```
return entityResource.search(args.term, 'Document', args.searchFrom, args.canceler, args.dataTypeKey)
       .then(data => {
               data.forEach(function (item) {
                     return searchResultFormatter.configureContentResult(item);
               });
               return data;
      });
```

Reply:
I agree @bjarnef , that would be more consistant.